rxandroidble/src/main/java/com/polidea/rxandroidble2/internal/scan/InternalScanResultCreator.java

```diff
@@ -13,7 +13,6 @@
 import androidx.annotation.RestrictTo;
 import com.polidea.rxandroidble2.ClientScope;
 import com.polidea.rxandroidble2.internal.RxBleLog;
-import com.polidea.rxandroidble2.internal.util.UUIDUtil;
```

Comment:
Why did you removed this import?

Reply:
Because you can’t suppress warnings on imports, so you have to use the full namespace path and remove the import.

Reply:
(This is fixed in Java 9)